README.md

```diff
@@ -41,6 +41,21 @@ generator jsonSchema {
 }
 ```
 
+Additional options
+
+```prisma
+generator jsonSchema {
+  provider = "prisma-json-schema-generator"
+  keepRelationScalarFields = true
```

Comment:
Do you have any preferences as to whether the option should be documented here as `true` or `"true"`. The prisma generator appears to parse everything as a string anyway.  But I did see another project that used boolean values without the quotes here

Reply:
I would prefer quotes here. Maybe the team behind Prisma decides to correctly parse boolean values in future releases, which actually would break the plugin usage.

Reply:
good point! i didn't think of that.